content/en/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md

```diff
@@ -13,56 +13,181 @@ variables and then consume them using MicroProfile Config.
 
 ## {{% heading "prerequisites" %}}
 
-### Creating Kubernetes ConfigMaps & Secrets
+Before starting, make sure you have the following prerequisites:
+
+1. **Basic understanding of Kubernetes concepts**
+
+   If you're new to Kubernetes, it's essential to familiarize yourself with its core concepts. You can get started by exploring the [Kubernetes Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/) tutorial.
+
+2. **A running Kubernetes cluster**
+
+   Set up a Kubernetes cluster according to your platform:
+
+   - For **Mac** users, you can use [Docker Desktop](https://www.docker.com/products/docker-desktop) to run a Kubernetes cluster locally.
+
+   - For **Windows** users, Docker Desktop also supports Kubernetes on Windows.
+
+   - For **Linux** users, you can use [Minikube](https://minikube.sigs.k8s.io/docs/start/) or set up a Kubernetes cluster on a cloud provider like [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine) or [Amazon Elastic Kubernetes Service (EKS)](https://aws.amazon.com/eks/).
+
+3. **Java development environment set up on your local machine**
+
+   Ensure you have Java installed. You can check by running `java -version` in your terminal. If not installed, follow these steps:
+
+   - **Mac**:
+     ```bash
```

Comment:
```suggestion

     ```bash
```
Adding a blank line before this code block will display the following text in its regular format.

Reply:
@windsonsea Thanks for the suggestion. As this is my first PR so I'm not sure how to proceed further with the suggestion. Should I commit it directly? as there is option for "Commit suggestion" under your suggestion box or should I make changes to my branch repo again ? Could you please guide me?

Reply:
Checkout the `updatedtutorial` branch on your local repo.
Revise the contents according to the suggestions.
Then `git commit --amend` to your own clone.
Github will refresh your PR automatically.

Reply:
@tengqm Thanks for the help. 
@windsonsea I have amended the suggestion. What to do next?